bin/install.sh

```diff
@@ -436,6 +436,14 @@ install_vars_export() {
     done
 }
 
+# TODO(e-carlin): remove when f36 is deployed (https://github.com/radiasoft/download/issues/379)
+install_version_fedora_is_36() {
```

Comment:
When backporting, it's usually better to make the older version, e.g. 

```sh
install_version_fedora_lt_36() {
    if (( install_version_fedora < 36 )); then
        return 0
    fi
    return 1
}
```

You don't have to remember to take this code out when going to 38, for example.

Reply:
a6bfa33